.gitignore

```diff
@@ -0,0 +1,26 @@
+*~
+*.swp
+*.cache
+*.aux
+*.log
+*.pdf
+*.dvi
+*.lof
+*.toc
+*.pdftex_t
+*.out
+*.bak
+.#*
+.panfs.*
+*.svg
+*.bbl
+*.blg
+*.lot
+*.exo
+*.mod
+/build*
+/install
+/doc/browser/doc/html/
+/doc/doxygen/html/
+.ycm_extra_conf.py
+.ycm_extra_conf.pyc
```

Comment:
I am not a fan on having so many things in the `.gitignore`, I would prefer a minimal or no `.gitignore`

Reply:
Rationale?

Reply:
Because I like to mindlessly do `git add .` and where there is a lot of things in the `.gitignore` important files can be ignored. Just think about the image for the paper that was a pdf and that you forgot to push because it was in your `.gitignore`